lyricurator.py

```diff
@@ -23,6 +23,8 @@ def main():
             tweet = get_tweet_string(artist, song_name, lyrics)
             print(tweet)
             print("\nFull song lyrics can't be tweeted")
+        elif  CONSOLE_ARGS.notwitter and CONSOLE_ARGS.hotsongs:            
+            get_hot_songs()
```

Comment:
Same as last comment. after you return the array `hot_songs`. you can do something like this here
```
hot_songs = get_hot_songs()
print(*hot_songs, sep = "\n")
```